src/components/visualizations/chart/legend/LegendList.tsx

```diff
@@ -1,15 +1,22 @@
-// (C) 2007-2018 GoodData Corporation
+// (C) 2007-2019 GoodData Corporation
 import * as React from 'react';
+import { injectIntl } from 'react-intl';
 import LegendItem from './LegendItem';
 import { LEGEND_AXIS_INDICATOR, LEGEND_SEPARATOR } from './helpers';
-import { injectIntl } from 'react-intl';
+import { ISeriesItem } from '../../../../interfaces/Config';
 
-const LegendAxisIndicator = ({ axis, width, intl }: any) => {
+const LegendAxisIndicator = ({ labelKey, data, width, intl }: any) => {
```

Comment:
try to define types

Reply:
Maybe it will be fine to move this to separate file and from functional component make pure component.